public/app.d.ts

```diff
@@ -1428,32 +1428,200 @@ declare namespace JMap {
       }
 
       /**
-       * **JMap.Application.Layer.startThematicEdition**
+       * **JMap.Application.Layer.Edition**
        * 
-       * If layer has no thematic, do nothing, just log a warning message in the console.
-       * 
-       * @throws If layer is not found or layerId is correspond to a layer group
-       * @example ```ts
-       * 
-       * // Will edit the layer's id="3" thematics
-       * JMap.Application.Layer.startThematicEdition(3)
-       * ```
+       * You can manage the layer edition panel here.
        */
-      function startThematicEdition(layerId: JId): void
+      namespace Edition {
 
-      /**
-       * **JMap.Application.Layer.stopThematicEdition**
-       * 
-       * If we are editing thematics of a layer, will stop edition and show the layer tree panel, 
-       * Else log a warning message to tell we are not editing thematics.
-       * 
-       * @example ```ts
-       * 
-       * // will return to the layer tree panel
-       * JMap.Application.Layer.stopThematicEdition()
-       * ```
-       */
-      function stopThematicEdition(): void
+        /**
+         * **JMap.Application.Layer.Edition.DynamicFilter**
+         * 
+         * You can manage layer dynamic filter panel and dialog here.
+         */
+        namespace DynamicFilter {
+
+          /**
+           * **JMap.Application.Layer.Edition.DynamicFilter.openPanel**
+           * 
+           * Open edition panel, with "dynamic-filter" tab activated, for the given layer
```

Comment:
"Opens _**the**_ edition panel"

(mettre un "s" à tous les verbes de la troisième personne du singulier dans toutes les sections plus bas aussi)